CMakeLists.txt

```diff
@@ -22,19 +20,27 @@ endif()
 if(NOT BUILD_COMM_TCP_PLAIN)
     set(BUILD_COMM_TCP_PLAIN FALSE)
 endif()
+if(NOT BUILD_COMM_TCP_TLS)
+    set(BUILD_COMM_TCP_TLS FALSE)
+endif()
 
 if(NOT CONCORD_LOGGER_NAME)
     set(CONCORD_LOGGER_NAME "concord")
 endif()
 
+
+
+
+
+
```

Comment:
nit: this is a lot of added whitespace - is it all needed?

Reply:
removed